package.json

```diff
@@ -33,6 +33,7 @@
     "react-focus-on": "^3.8.1",
     "react-i18next": "^13.0.1",
     "react-idle-timer": "^5.7.2",
+    "react-markdown": "^8.0.7",
```

Comment:
Any reason to use `react-markdown` isntead of `markdown-to-jsx`? YDG and SJ are both using `markdown-to-jsx`.
https://probablyup.com/markdown-to-jsx/